packages/algoliasearch/src/builds/browser.ts

```diff
@@ -319,10 +319,26 @@ export default function algoliasearch(
           },
         });
       },
+      initPersonalization: () => (
+        clientOptions?: InitPersonalizationOptions
+      ): PersonalizationClient => {
+        return createPersonalizationClient({
+          ...commonOptions,
+          ...clientOptions,
+          methods: {
+            getPersonalizationStrategy,
+            setPersonalizationStrategy,
+          },
+        });
+      },
       initRecommendation: () => (
```

Comment:
does this work like this?
```suggestion
      initRecommendation: (args)  => {warn(...) return this.initPersonalization(args)}
```

(we don't have a `deprecate` in this codebase at the moment I think, it would fit nicely otherwise

Reply:
Not really but I went with 95cf1089c7af9dbc2772c74d89357ce2d1367608.